studio/utils/result.ts

```diff
@@ -0,0 +1,9 @@
+export type Result<T, E> = { ok: true; value: T } | { ok: false; error: E };
```

Comment:
add blockComments to explain what these are? 

should we keep them under components/salariesData/utils or do you know they'll be used elsewhere?

Reply:
Added some documentation now :+1: This is a very generic type that can be used for any return type with a possible failure state. It's difficult to know if it will be used elsewhere, but I think placing it here will encourage its usage.